### ISSUE TITLE
[HIGHEST URGENCY!] VOLUNTEER PORTAL BROKEN - blank page when linked

### ISSUE THREAD

**Issue by dreazee** [SDRefugeeTutoring/sdrt-custom-functions#112]
@JasonTheAdams Oh hi! So... Volunteer Portal isn't showing up!!! 
![Screenshot (102)](https://user-images.githubusercontent.com/47679296/217155235-9ba8a825-5879-433f-9a65-c962fa8b1b3b.png)

**Comment by JasonTheAdams** [comment]
This is resolved in ef23c2c.

WordPress was doing something rather unexpected with the `get_terms` function which is used to retrieve the trimesters. If multiple trimesters we retrieved but one didn't have any posts, then it would omit that trimester but change the shape of the data. This fix simply makes sure that the data is always the expected shape, so this shouldn't be a problem in the future, and we can continue to make Trimesters as needed.

In short, making future Trimesters shouldn't be an issue, and now it's not. 😄
